### PR TITLE
new test file

### PR DESCRIPTION
**Pull request by krissbec**
Please review this file!

### CODE REVIEW

File: collab-file.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Want to 
+
```
Comment (bestksbever) [inline collab-file.md:2]: You should write something here!

File: collab-file.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+test how to merge/pull request this
```
Comment (bestksbever) [inline collab-file.md:3]: This should change!

**Review by bestksbever — CHANGES_REQUESTED** [review]
I have some changes you need to do

File: collab-file.md (hunk 1 above)
Comment (krissbec) [reply collab-file.md:2]: Got it!, will do!

File: collab-file.md (hunk 2 above)
Comment (krissbec) [reply collab-file.md:3]: Okay!

**Pull request merged**